package.json

```diff
@@ -1,12 +1,12 @@
 {
   "private": "true",
   "dependencies": {
-    "benchmark": "~1.0.0",
-    "microtime": "~1.2.0",
-    "pulp": "~4.3.0"
+    "benchmark": "^2.1.0",
+    "microtime": "^2.1.1",
+    "pulp": "^9.0.1"
   },
   "scripts": {
-    "postinstall": "pulp dep install",
+    "postinstall": "npm install -g bower && bower install",
```

Comment:
We shouldn't be installing anything globally in the `postinstall` script. Perhaps we could try depending on bower in the `dependencies` section instead? npm scripts set up your path to include executables from locally installed packages so this ought to work I think.